rotkehlchen/chain/ethereum/oracles/saddle.py

```diff
@@ -78,9 +78,13 @@ def query_current_price(self, from_asset: Asset, to_asset: Asset) -> Price:
         try:
             return self.get_price(
                 from_asset=from_asset.resolve_to_crypto_asset(),
-                to_asset=to_asset.resolve_to_crypto_asset(),
+                to_asset=to_asset.resolve_to_asset_with_oracles(),
                 block_identifier='latest',
             )
-        except UnknownAsset as e:
+        except (UnknownAsset, WrongAssetType) as e:
             # Means that either from_asset or to_asset is not a crypto asset
-            raise PriceQueryUnsupportedAsset(e.asset_name) from e
+            if isinstance(e, WrongAssetType):
```

Comment:
hmm this seems like a really silly thing to do at every place where we handle both exceptions. Why not give the name to `WrongAssetType` or the identifier to `UnknownAsset` ?

Reply:
Changed the UnsupportedAsset and UnknownAssets fields to be the same as WrongAssetType because otherwise we would have the same issue